src/e2e-test/resources/testdata/GCS_DATATYPE_TEST_SPANNER.csv

```diff
@@ -0,0 +1,5 @@
+id,name,yearofbirth,isdeleted,email,createddate,revenue,points,longdatatype,doubledatatype,date,BytesData
```

Comment:
As you have created new csv file for spanner sink,
check if you can also add remaining datatypes`float_inf,float_nan,decimal` from `GCS_DATATYPE_TEST_2.csv to `GCS_DATATYPE_TEST_SPANNER.csv`
so that all the datatypes can be covered in single scenario.